arches_component_lab/src/arches_component_lab/widgets/ResourceInstanceMultiSelectWidget/components/ResourceInstanceMultiSelectWidgetEditor.vue

```diff
@@ -64,6 +64,10 @@ watch(
 
 const resourceResultsCurrentCount = computed(() => options.value.length);
 
+onMounted(async () => {
```

Comment:
if you're keeping `before-show`, you may not need this? I def won't block on this either way though.

Reply:
It's still needed; the values need to be available on component load - the before-show is too late.  Removing this causes the chips to be empty.